docs/resources/live_notification_cofiguration.md

```diff
@@ -0,0 +1,60 @@
+---
+subcategory: "Live"
+layout: "huaweicloud"
+page_title: "HuaweiCloud: huaweicloud_live_notification_cofiguration"
+description: |-
+  Manages a notification cofiguration resource within HuaweiCloud.
+---
+
+# huaweicloud_live_notification_cofiguration
+
+Manages a notification cofiguration resource within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable "domain_name" {}
+variable "url" {}
+
+resource "huaweicloud_live_notification_cofiguration" "test" {
+  domain_name = var.domain_name
+  url         = var.url
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the resource.
+  If omitted, the provider-level region will be used.
+  Changing this parameter will create a new resource.
+
+* `domain_name` - (Required, String, ForceNew) Specifies the ingest domain name to which the notification cofiguration
+  belongs.
+  Changing this parameter will create a new resource.
+
+* `url` - (Required, String) Specifies the callback URL, which must start with `http://` or `https://`.
+
+* `auth_sign_key` - (Optional, String) Specifies the authentication key.
+  The valid length is `32` to `128` characters. This parameter can be left blank.
```

Comment:
```suggestion
  The valid length is `32` to `128` characters.
```

Reply:
`Optional` already indicates that this field can be left unconfigured, so no special instructions are needed.